src/main/java/org/stellar/sdk/ManageOfferOperation.java

```diff
@@ -1,164 +1,14 @@
 package org.stellar.sdk;
 
-import org.stellar.sdk.xdr.CreateAccountOp;
-import org.stellar.sdk.xdr.Int64;
-import org.stellar.sdk.xdr.ManageOfferOp;
-import org.stellar.sdk.xdr.OperationType;
-import org.stellar.sdk.xdr.Uint64;
-
-import java.math.BigDecimal;
-import java.math.MathContext;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
 /**
- * Represents <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html#manage-offer" target="_blank">ManageOffer</a> operation.
- * @see <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html" target="_blank">List of Operations</a>
+ * @deprecated Use {@link ManageSellOfferOperation}
```

Comment:
Should I deprecate a class like this?

Reply:
See the comment under `CreatePassiveOffer`.